people-ops/international-compensation.html.md

```diff
@@ -0,0 +1,75 @@
+---
+title: International compensation
+meta_description: >
+  Nebulab strives to hire the best fit-talents both in culture and skill-set. Our international
+  compensation policy aims to pay the right salary for them. Find out how we handle this!
+---
+
+> Note: The work here is inspired by [Gitlab's approach to international compensation](https://about.gitlab.com/handbook/total-rewards/compensation/#paying-local-rates),
+> which we believed to be a good starting point.
+
+We hire people in different regions and countries. Every market is different, and it can be
+challenging to be competitive and fair towards other team members in making an offer for the same
+role but in a different region. The best way to deal with that is by introducing a Location factor.
+
+### Why we pay local salaries
+
+The key argumentations on why we value this policy as a critical success factor, both in hiring
```

Comment:
`The key argumentations on why`

`The reasons why` sounds better to me

Reply:
yeah I agree